_posts/2019-05-24-server-rendering-responsively.md

```diff
@@ -49,13 +49,15 @@ While only a few of the properties in this example _are_ specifying an array of
 breakpoints, all of those properties _can_ take an array for different breakpoints. As developers, we love this
 experience. We can apply subtle differences to components across breakpoints with very little code and effort.
 
-We use `styled-system` extensively within [our design system](https://palette.artsy.net/). You can
+We use [`styled-system`](https://styled-system.com) extensively within
```

Comment:
For future reference, you can use “reference” links in markdown instead of having to repeat full inline URLs.

Reply:
Neat! TIL.